src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
```

Comment:
[CODE STYLE] - Variable naming like `stylesObject` could be improved for better readability. Consider a more descriptive name that reflects the purpose of the object, such as `styleRules` or `cssProperties`.